lexik/jwt-authentication-bundle/2.3/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Lexik\\Bundle\\JWTAuthenticationBundle\\LexikJWTAuthenticationBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%"
+    },
+    "env": {
+        "JWT_PRIVATE_KEY_PATH": "%kernel.project_dir%/var/jwt/private.pem",
```

Comment:
I wasn't comfortable with putting this in  the`app/config` directory of the SE as this is not something editable, but I agree and it feels right to me now that this dir is called `etc`.
Updated to `%ETC_DIR%/jwt/*.pem`.